### PR TITLE
Add container env as tags docs

### PR DESCRIPTION
**Pull request by ahmed-mez**
### What does this PR do?
Document this https://github.com/DataDog/datadog-agent/pull/9054

### Preview

https://docs-staging.datadoghq.com/ahmed/container-env-as-tags/agent/kubernetes/tag

### Additional Notes
Do not merge before releasing Agent 7.32

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by davidor — APPROVED** [review]

File: content/en/agent/kubernetes/tag.md [hunk 1]
```diff
@@ -305,6 +305,49 @@ kubernetes_namespace_labels_as_tags:
...
+DD_CONTAINER_ENV_AS_TAGS='{"<ENV_VAR>": "<TAG_KEY>"}'
+```
+
+For example, you could set up:
+
+```shell
+DD_CONTAINER_ENV_AS_TAGS='{"app":"kube_app"}'
+```
+
+**Note**: Custom metrics may impact billing. See the [custom metrics billing page][1] for more information.
```
Comment (alai97) [inline content/en/agent/kubernetes/tag.md:327]:
```suggestion
**Note**: Custom metrics may impact billing. See [Custom Metrics Billing][1] for more details.
```

File: content/en/agent/kubernetes/tag.md [hunk 2]
```diff
@@ -305,6 +305,49 @@ kubernetes_namespace_labels_as_tags:
...
+{{< tabs >}}
+{{% tab "Containerized Agent" %}}
+
+To extract a given environment variable `<ENV_VAR>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following environment variable to the Datadog Agent:
+
+```shell
+DD_CONTAINER_ENV_AS_TAGS='{"<ENV_VAR>": "<TAG_KEY>"}'
+```
+
+For example, you could set up:
```
Comment (alai97) [inline content/en/agent/kubernetes/tag.md:321]:
```suggestion
For example:
```

File: content/en/agent/kubernetes/tag.md [hunk 3]
```diff
@@ -305,6 +305,49 @@ kubernetes_namespace_labels_as_tags:
...
+{{% tab "Agent" %}}
+
+To extract a given environment variable `<ENV_VAR>` and transform it as a tag key `<TAG_KEY>` within Datadog, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
+
+```yaml
+container_env_as_tags:
+  <ENV_VAR>: <TAG_KEY>
+```
+
+For example, you could set up:
```
Comment (alai97) [inline content/en/agent/kubernetes/tag.md:340]:
```suggestion
For example:
```

**Review by alai97 — APPROVED** [review]
Looks good pending these nits!

**Comment by alai97** [comment]
The Agent 7.32 release is scheduled to be released in the next four weeks.

**Comment by ahmed-mez** [comment]
@alai97 Agent 7.32 has been released, we can merge this!

**Pull request merged**